VSGitDiff/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="VSGitDiff.Jonathan Galentine.d5d61ff8-f8ae-4a06-92be-b0bbedc36e2f" Version="1.2" Language="en-US" Publisher="Jonathan Galentine" />
+        <Identity Id="VSGitDiff.Jonathan Galentine.d5d61ff8-f8ae-4a06-92be-b0bbedc36e2f" Version="1.3" Language="en-US" Publisher="Jonathan Galentine" />
```

Comment:
Changes here (including removal of now unnecessary MPF dependency) are [based off of recommendations in this MS dev blog article on upgrading extensions to VS 2019](https://devblogs.microsoft.com/visualstudio/how-to-upgrade-extensions-to-support-visual-studio-2019/).